src/test/java/com/google/api/codegen/transformer/go/GoSurfaceNamerTest.java

```diff
@@ -49,4 +50,17 @@ public void testOperationName() {
     Truth.assertThat(namer.getAndSaveOperationResponseTypeName("CreateStuffOperation"))
         .isEqualTo("CreateStuffOperationHandle");
   }
+
+  @Test
+  public void testPutDocMethodName() {
+    GoSurfaceNamer namer = new GoSurfaceNamer("cloud.google.com/go/gopher/apiv1");
+    Truth.assertThat(namer.putDocMethodName("Create", "creates things", VisibilityConfig.PUBLIC))
+        .isEqualTo("Create creates things");
+    Truth.assertThat(namer.putDocMethodName("Create", "Creates things", VisibilityConfig.PUBLIC))
+        .isEqualTo("Create creates things");
+    Truth.assertThat(
+            namer.putDocMethodName(
+                "Create", "Create is used to create things", VisibilityConfig.PUBLIC))
```

Comment:
How about also 
`"Create" vs "creAte is used to create things"` and 
`"CreateThing" vs "creatething is used to create things"`